build.py

```diff
@@ -1588,8 +1629,10 @@ def __check_tools(self, opts):
         print_debug("wget: %s" % (self.wget,))
 
         self.nuget = opts.nuget_path
-        if not os.path.exists(self.nuget):
-            print_log("Could not find nuget: %s" % (self.nuget,))
+        if opts.nuget_path:

```

Comment:
at this point I really think we don't need this. Let's remove the option to pass in the nuget path